BEIMA.Client/src/shared/DropDown/FilledDropDown.js

```diff
@@ -0,0 +1,19 @@
+import { Dropdown } from 'react-bootstrap';
+
+const FilledDropDown = ({items, selectFunction, buttonStyle}) => {
```

Comment:
An idea that can happen later, but we can change the Building input to be a drop down and use this shared component to populate the available buildings. Nice work!

Reply:
All I did was rip off your idea, but thanks